todo_app/e2e_test.py

```diff
@@ -57,8 +84,6 @@ def test_update_to_done_new_item(driver, app_with_temp_board):
      driver.get('http://localhost:5000/')
      submit_update_to_done = '/html/body/p[18]/input[2]'
```

Comment:
These paths are very specific and make it both hard to check/adjust and also very fragile to changes.  Once we fix the fact it's not correctly overriding the environment variables, this doesn't find the right input. 
A much cleaner and more reliable xpath for this update to done might be more like `'//input[@type = "submit" and @value="Update to done"]'`

Reply:
Updated,thank you